designcompose/src/main/java/com/android/designcompose/DesignView.kt

```diff
@@ -954,7 +951,7 @@ internal fun DesignDocInternal(
             LaunchedEffect(docId) { designComposeCallbacks?.docReadyCallback?.invoke(docId) }
             CompositionLocalProvider(LocalDesignIsRootContext provides DesignIsRoot(false)) {
                 DesignView(
-                    modifier.semantics { sDocId = docId },
+                    modifier.semantics { sDocRenderStatus = docRenderStatus },
```

Comment:
How will this part work with in the rustlayout branch?

Reply:
I don't know, and so I'm basically punting that. 

Basically, we don't have a solution for getting the test assertions to work yet. But I'm pretty sure I can get screenshot tests working either way.

I still think this is a good change. The changes that I made to the actual tests will persist to the rustlayout branch. Basically we'll need to figure out how to get the semantics applied in rustlayout, but the tests themselves won't need to change.

Reply:
Ok that's fine, you'll just have to deal with some conflicts merging to the feature branch or merging the feature branch back to main.

Reply:
Yep